### DISCUSSION TITLE
Error

### DISCUSSION THREAD

**Discussion by dafunks** [Luois45/claim-free-steam-packages#26] [General]
```
Downloaded repo package list with 1 free packages.
Out of 1 known free packages, all are already activated. Skipping activation phase.
Waiting 2 hours to check for new free packages.
```

For some reason this is happening. There is no way I have finished as yesterday I had only 2% done and had only activated around 500. It seems something has gone wrong.

**Comment by louisa-uno** [comment]
Created the issue #27 for this error.
It is happening because the workflow run for updating the package list keeps failing.

**Comment by dafunks** [comment]
Thank you! All working now. Great work :)
